File: Ejercicios_etapa_1/00_Javascript/Ejercicio_1/prueba.test.js
import { describe, it, expect } from "vitest"
import { edadEnDias } from "./index.js" 

describe("edadEnDias", () => {
  it("debe calcular correctamente la edad en días para 1 año", () => {
    expect(edadEnDias(1)).toBe(365)
  })

  it("debe calcular correctamente la edad en días para 0 años", () => {
    expect(edadEnDias(0)).toBe(0)
  })

  it("debe calcular correctamente la edad en días para 10 años", () => {
    expect(edadEnDias(10)).toBe(3650)
  })
})

File: Ejercicios_etapa_1/00_Lua/Ejercicio_1/prueba.test.lua
package.path = "./?.lua;" .. package.path

local edadModule = require("edadEnDias")
local edadEnDias = edadModule.edadEnDias

describe("edadEnDias", function()
    it("debe calcular 365 para 1 año", function()
        assert.are.equal(365, edadEnDias(1))
    end)

    it("debe calcular 0 para 0 años", function()
        assert.are.equal(0, edadEnDias(0))
    end)

    it("debe calcular 3650 para 10 años", function()
        assert.are.equal(3650, edadEnDias(10))
    end)

    it("debe calcular 9490 para 26 años", function()
        assert.are.equal(9490, edadEnDias(26))
    end)
end)
